src/docker.test.ts

```diff
@@ -68,7 +69,9 @@ describe("Docker images", (): void => {
     }
     await docker.saveDockerImages();
 
-    expect(core.getInput).nthCalledWith(1, "key", { required: true });
+    expect(core.getInput).nthCalledWith<[string, InputOptions]>(1, "key", {
```

Comment:
Multiple instances: can we fit all of the arguments on the line below, or does Prettier force this awkward formatting?

Reply:
I attempted your formatting suggestion but, alas, Prettier objected.